pkg/storage/chunk/chunk.go

```diff
@@ -94,15 +101,17 @@ const BatchSize = 12
 // Batch is intended to be small, and passed by value!
 type Batch struct {
 	Timestamps [BatchSize]int64
-	Values     [BatchSize]float64
+	// Values stores float values related to this batch if ValueType is chunkenc.ValFloat.
+	Values [BatchSize]float64
 	// PointerValues store pointers to non-float complex values like histograms, float histograms or future additions.
 	// Since Batch is expected to be passed by value, the array needs to be constant sized,
 	// however increasing the size of the Batch also adds memory management overhead. Using the unsafe.Pointer
 	// combined with the ValueType implements a kind of "union" type to keep the memory use down.
-	PointerValues [BatchSize]unsafe.Pointer
-	ValueType     chunkenc.ValueType
-	Index         int
-	Length        int
+	PointerValues   [BatchSize]unsafe.Pointer
+	FloatHistograms [BatchSize]*histogram.FloatHistogram
```

Comment:
we don't need FloatHistograms  anymore, based on the  ValueType PointerValues can hold it

Reply:
You are right, that was a mistake. Thank you